net/rp-pppoe/Makefile

```diff
@@ -1,5 +1,5 @@
 #
-# Copyright (C) 2006-2016 OpenWrt.org
+# Copyright (C) 2006-2022 OpenWrt.org
```

Comment:
Why?

Reply:
I just think this version is valid for an update.